services/PA-accounts-service/src/data-storage/data-storage.service.ts

```diff
@@ -35,11 +35,11 @@ export class DataStorageService {
       order: { created: 'DESC' },
     });
     if (!data || data.length === 0) {
-      const errors = { Data: ' not found' };
-      throw new HttpException({ errors }, HttpStatus.NOT_FOUND);
+      throw new HttpException('', HttpStatus.NOT_FOUND);
     }
 
     data[0].data = this.cryptr.decrypt(data[0].data);
-    return JSON.stringify(data[0].data);
+
+    return data[0].data;
```

Comment:
This extra 'stringification' isn't really necessary to do explicitly here..

But I think that NestJS will do it for us anyway when we do:
`return { data };`

So no _real_ win in the end.